### PR TITLE
CI: Support integration tests testing for WASM

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
This `PR` adds support for `wasm` `integration_tests` testing at the `CI`.

### CODE REVIEW

**Review by certik — APPROVED** [review]
I think this is fine. Thanks!

File: integration_tests/CMakeLists.txt [hunk 1]
```diff
@@ -69,7 +69,19 @@ macro(RUN)
         endif()
 
         if ((LFORTRAN_BACKEND STREQUAL "wasm"))
-            add_test(${name} node --experimental-wasm-bigint ${CURRENT_BINARY_DIR}/${name}.js)
+            find_program(WASM_EXEC_RUNTIME node)
+            execute_process(COMMAND "${WASM_EXEC_RUNTIME}" --version
+                            OUTPUT_VARIABLE WASM_EXEC_VERSION
+                            OUTPUT_STRIP_TRAILING_WHITESPACE)
+            string(COMPARE GREATER_EQUAL "${WASM_EXEC_VERSION}"
```
Comment (Shaikh-Ubaid) [inline integration_tests/CMakeLists.txt:76]: **On Line 76:**

I used `GREATER_EQUAL` for knowing if the current `node` version is `>= 16`. As per `cmake` `docs` https://cmake.org/cmake/help/latest/command/string.html#comparison, the ` GREATER_EQUAL` option is added in `cmake` version `3.7`. The current required `cmake` version as per `integration_tests/CMakeLists.txt` is `3.5`. Also, the current required `cmake` version as per `CMakeLists.txt` (present in the root directory) is `3.10`.

@certik , please, could you possibly share if we shall/could update the `cmake` version in `integration_tests/CMakeLists.txt` to `3.10`:
- to make it in sync with the `CMakeLists.txt` (from root directory)
- to support the string compare option `GREATER_EQUAL`

Comment (certik) [reply integration_tests/CMakeLists.txt:76]: Yes, go ahead and update it to 3.10.

**Review by certik — APPROVED** [review]
Looks good to me.

**Pull request merged**